mainwindow.cpp

```diff
@@ -23,6 +24,7 @@ MainWindow::MainWindow(QWidget *parent) :
     resizer->setBrush(QColor(64, 64, 64));
     resizer->setMinSize(QSizeF(30, 30));
     resizer->setTargetSize(item->boundingRect().size());
+    resizer->setHandlersIgnoreTransformations(true);
```

Comment:
It'd be great to add some kind of checkbox to toggle this value on and off, I guess